src/trusted/service_runtime/nacl_app_thread.c

```diff
@@ -147,13 +426,12 @@ void NaClAppThreadTeardown(struct NaClAppThread *natp) {
   NaClLog(3, " freeing thread object\n");
   NaClAppThreadDelete(natp);
   NaClLog(3, " NaClThreadExit\n");
+
   NaClThreadExit();
-  NaClLog(LOG_FATAL,
-          "NaClAppThreadTeardown: NaClThreadExit() should not return\n");
+  NaClLog(LOG_FATAL, "NaClAppThreadTeardown: NaClThreadExit() should not return\n");
```

Comment:
It's a bit odd because the number refers to number of -v needed to see the message, with macros for the special cases. I am aware it's not ideal, and I'm still trying to figure out of there's a better way. This would definitely be a change for a seperate PR, though.